fastlane_bot/events/exchanges/balancer.py

```diff
@@ -83,4 +83,10 @@ async def get_tkn_n(self, address: str, contract: Contract, event: Any, index: i
         pool_balances = await contract.caller.getPoolTokens(address)
         tokens = pool_balances[0]
         token_balances = pool_balances[1]
-        return token_balances[index]
\ No newline at end of file
+        return token_balances[index]
+
+    def get_pool_function(self, factory_contract: Contract):
+        """
+        This function is unused for Carbon.
+        """
+        pass
```

Comment:
Either this function is not needed to begin with, or it is needed, and then attempting to call it and make use of the returned value will fail, since it doesn't return anything.

Reply:
Not relevant any more